integreat_cms/api/v3/social_media_headers.py

```diff
@@ -41,13 +42,25 @@
 
 def get_excerpt(content: str) -> str:
     """
-    Correctly escapes, truncates and normalizes the content of the page to display in a search result
+    Correctly escapes, truncates and normalizes the content of the page to display in a search result.
+    Then apply some crazy string operations to handle several edge cases.
```

Comment:
I personally feel like this insertion doesn't really add any information: I'd either leave it as it is or specify which operations we added to handle what edge-case.